pkg/config/expand.go

```diff
@@ -180,20 +180,22 @@ func (mod *Module) addListValue(settingName string, value string) error {
 		errorMessages["appendToNonList"], mod.ID, settingName)
 }
 
-// This function matches input variables in a "using" module to output values
+// useModule matches input variables in a "using" module to output values
 // from a "used" module. It may be used iteratively to successively apply used
 // modules in order of precedence. New input variables are added to the using
 // module as Toolkit variable references (in same format as a blueprint). If
 // the input variable already has a setting, it is ignored, unless the value is
 // a list, in which case output values are appended and flattened using HCL.
-// mod: "using" module as defined above
-// useMod: "used" module as defined above
-// useModGroupID: deployment group ID to which useMod belongs
-// modInputs: input variables as defined by the using module code
-// useOutputs: output values as defined by the used module code
-// settingsToIgnore: a list of module settings not to modify for any reason;
 //
-//	typical usage will be to leave explicit blueprint settings unmodified
+//	mod: "using" module as defined above
```

Comment:
I am trying to match what I perceive to be [godoc style](https://tip.golang.org/doc/comment) and something that is treated the same by "go fmt" in both 1.18 and 1.20.